property.sql

```diff
@@ -0,0 +1,37 @@
+CREATE TABLE IF NOT EXISTS `properties` (
+	`id` INT NOT NULL AUTO_INCREMENT,
+	`name` VARCHAR(255) NOT NULL,
+	`interior` VARCHAR(255) NOT NULL,
+	`furnished` BOOLEAN NOT NULL,
+	`owners` JSON,

```

Comment:
Why have this field if we have a property_owners table?

Reply:
I think i forgot to remove that my bad